react/components/Slider.js

```diff
@@ -402,25 +335,23 @@ class Slider extends Component {
   }
 
   onMouseMove = e => {
-    const { easing, loop, currentSlide, draggable, cursorOnMouseDown } = this.props
+    const { currentSlide, draggable, cursorOnMouseDown, easing } = this.props
 
     e.preventDefault()
     if (this.pointerDown && draggable) {
       // TODO prevent link clicks
 
       this.drag.endX = e.pageX
 
-      const computedCurrentSlide = loop ? currentSlide + this.perPage : currentSlide
-      const currentOffset = computedCurrentSlide * (this.selectorWidth / this.perPage)
+      const currentOffset = currentSlide * (this.selectorWidth / this.perPage)
       const dragOffset = (this.drag.endX - this.drag.startX)
-      const offset = currentOffset - dragOffset
+      const offset = (currentOffset - dragOffset) / this._sliderFrame.current.getBoundingClientRect().width * -100
 
       setStyle(this._sliderFrame.current, {
         cursor: cursorOnMouseDown,
-        WebkitTransition: `all 0ms ${easing}`,
-        transition: `all 0ms ${easing}`,
-        transform: `translate3d(${offset * -1}px, 0, 0)`,
-        WebkitTransform: `translate3d(${offset * -1}px, 0, 0)`
+        ...getStylingTransition(easing),
+        transform: `translate3d(${offset}%, 0, 0)`,
+        WebkitTransform: `translate3d(${offset}%, 0, 0)`
```

Comment:
why is the function `setTransformProperty` not being used here?

Reply:
oh, I see why. Why then, does not that function simply return the `transform` object? So you don't have to create a specific `webkitTransform`? (actually, I think `webkitTransform` is not even needed at this point, but I'm not sure)

Reply:
There is some old versions of some browsers that need this, so I thought that would be nice to let it here